js/sts-hotspot.js

```diff
@@ -0,0 +1,45 @@
+AFRAME.registerComponent('sts-hotspot', {
+  schema: {
+    show: { type: 'selector' },
+    sound: { type: 'selector' }
+  },
+
+  init: function () {
+    var self = this;
+
+    self.el.addEventListener('mouseenter', function (e) {
+      self.playHotspotSound();
+
+      // make popup visible
+      console.log('showing:',self.data.show);
+      self.data.show.setAttribute('material', { visible: true });
```

Comment:
upon close, when and where does the material's `visible` property get reset to `false`?

Reply:
It's set on the element itself in `index.html`  ie: `<a-entity visible="false">`